src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```
@@ -0,0 +1,153 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.UserRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void init() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User("myLogin", "myPassword", 21);
+    }
+
+    @Test
+    void register_User_Ok() {
+        registrationService.register(user);
+        assertThrows(UserRegistrationException.class, () -> registrationService.register(user));
```

why ok test should throw an exception